### ISSUE TITLE
Error: Subscript used on non-accessible variable in Helper\PEAutoRun\Drivers\main.au3

### ISSUE THREAD

**Issue by bezik46** [jmclaren7/windows-setup-helper#2]
Gives an error on boot

```
Line 10 (File "X:\Helper\PEAutoRun\Drivers\main.au3):
.
.
Error: Subscript used on non-accessible variable
```

**Comment by bezik46** [comment]
With help from AutoIT forums I got it solved

Inserted
```
If @error Then
	ConsoleWrite("Extended = " & @extended & @CRLF)
	Exit
EndIf
```

after 
```
$aFiles = _FileListToArrayRec(@ScriptDir, "*.inf", $FLTAR_FILES, $FLTAR_RECUR, $FLTAR_NOSORT, $FLTAR_FULLPATH)
```

No more error (if directory is empty)

**Comment by jmclaren7** [comment]
Thanks for pointing this out, fixed in commit dfcb5f4
